### PR TITLE
EVAKA-4184 basic fee thresholds editor

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

Add a basic fee thresholds editor for creating new fee thresholds.

Also changes InputField's wrapper div to be inline so that the component behaves more like an input by default.

### CODE REVIEW

File: frontend/src/employee-frontend/components/finance-basics/FeeThresholdsItemEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+              }))
+            }
+            placeholder="0"
+            symbol={'€'}
+            info={validationErrorInfo('minFee')}
+            hideErrorsBeforeTouched
+          />
+        </FixedSpaceColumn>
+      </RowWithMargin>
+      {showMaxFeeErrorsInfo ? <div>ERROR</div> : null}
```
Comment (hruoho) [inline frontend/src/employee-frontend/components/finance-basics/FeeThresholdsItemEditor.tsx:247]: Is this intentional?

Comment (jaahon) [reply frontend/src/employee-frontend/components/finance-basics/FeeThresholdsItemEditor.tsx:247]: 😂 No

**Comment by hruoho** [comment]
What about editing, should we implement the edit mode later in a separate task?

**Review by hruoho — APPROVED** [review]
Looks good to me despite that error thing 👍

**Comment by jaahon** [comment]
> What about editing, should we implement the edit mode later in a separate task?

Yeah there's a task for that. I wanted to get this first part out of the way and do the editing and copying later.

**Pull request merged**